### ISSUE TITLE
Has anyone used pytorch pretrained weight for training?

### ISSUE THREAD

**Issue by Wuziyi616** [jwyang/faster-rcnn.pytorch#636]
I used caffe vgg16 weight to train the network first and got an map around 0.75 on voc. But when I changed to use pytorch vgg16 weight, I only got an map around 0.7. I did change the image channel format from 'bgr' to 'rgb' according to README of this project, but it still didn't work well. So I wonder if it is because of the pretrained weight I used for training, since it's mentioned in README that caffe pretrained weight performs better than pytorch pretrained weight?

**Comment by Wuziyi616** [comment]
But an map gap of 0.05 is, well, I think different pretrained weight shouldn't make such a big difference. So I really want someone who have also used pytorch resnet pretrained weight to share your training results. Thanks!

**Comment by Wuziyi616** [comment]
I think I forgot about the normalize step... Sorry for that.
